README.md

```diff
@@ -1,4 +1,4 @@
-# Clickhouse Exporter for Prometheus (old clickhouse-server versions)
+# Clickhouse Exporter for Prometheus
```

Comment:
do you know about embedded prometheus endpoint in clickhouse-server?
look details https://clickhouse.com/docs/operations/server-configuration-parameters/settings#prometheus

this project  was released for old clickhouse-server versions which not have prometheus endpoints